rdrf/rdrf/services/io/defs/importer.py

```diff
@@ -515,6 +516,17 @@ def _create_registry_objects(self):
             config.consent_locked = registry_consent_locked
             config.save()
 
+        if "form_titles" in self.data:
+            titles = self.data["form_titles"]
+            for t in titles:
+                g = Group.objects.filter(name=t["group"]).first()
```

Comment:
What if the group doesn't exist in this system?
Shouldn't this be `get_or_create`?

Reply:
Updated.